### ISSUE TITLE
Indices for links to the same domain, so they don't look like dupes

### ISSUE THREAD

**Issue by tomaes** [demozoo/demozoo#300]
Example: https://demozoo.org/groups/490/

On this page, we have 2 pouet group links and 3 scene.org links. It looks a bit silly like that. Some sort of differentiation would be nice. Like Pouet (I) Pouet (II) ... scene.org (I) etc.

**Comment by nswaldman** [comment]
I'd like that as a default behaviour for having no custom link labels. Especially for pages like this:
https://demozoo.org/productions/3319/

No indices will help discerning between the party, final, video, etc. You'd want to be able to discern between those links.

**Comment by glennlunder** [comment]
That sort-of ties into #15 for custom labeling download links.

**Comment by tomaes** [comment]
I was thinking more about the group and external production links. Those don't necessarily need a lot of extra annotations. The download links (https://demozoo.org/productions/62/, and some other ARJ packed demos' pages suffer especially) need more info of course, but I suspected there was/is a ticket for that already. ;)

**Comment by nswaldman** [comment]
Agreed, but you might see why a solution to an overarching issue might solve your problem too. :)
